pkg/p2p/libp2p/headers.go

```diff
@@ -23,12 +23,11 @@ func sendHeaders(ctx context.Context, headers p2p.Headers, stream *stream) error
 	defer cancel()
 
 	if err := w.WriteMsgWithContext(ctx, headersP2PToPB(headers)); err != nil {
-		return fmt.Errorf("write message: %w", err)
+		return fmt.Errorf("send write message: %w", err)
```

Comment:
I am not sure why these log messages are changed.

Reply:
when debugging the tests the logged messages were exactly the same (both messages are the same for `sendHeader` and `handleHeaders`).

Reply:
This should have been annotated by the caller function. The consequence is that the log message in production will contain redundant message part `handle`, for example for handleHeaders here https://github.com/ethersphere/bee/blob/9c5bf9c2ea2908c767b7dde9a1c052b14b09e268/pkg/p2p/libp2p/libp2p.go#L342. If the problem is with the tests, it should be improved there, this way it has the impact on the clarity of log messages in production. Returned error annotations should not reference the function that is returned them, that is the responsibility of the caller function.

Reply:
cool. removed :+1: